From: Yauhen Kharuzhy <jekhor@gmail.com>
Date: Sat, 10 Mar 2012 16:06:05 +0300
Subject: JPWL: Link binaries with math library

Signed-off-by: Yauhen Kharuzhy <jekhor@gmail.com>
---
 applications/codec/Makefile.am |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/applications/codec/Makefile.am b/applications/codec/Makefile.am
index 17be3b8..1e31e46 100644
--- a/applications/codec/Makefile.am
+++ b/applications/codec/Makefile.am
@@ -78,7 +78,7 @@ JPWL_j2k_to_image_CPPFLAGS = \
 @LCMS1_CFLAGS@ \
 @LCMS2_CFLAGS@
 JPWL_j2k_to_image_CFLAGS =
-JPWL_j2k_to_image_LDADD = $(top_builddir)/libopenjpeg/jpwl/libopenjpeg_JPWL.la  @LCMS1_LIBS@ @LCMS2_LIBS@ @TIFF_LIBS@ @PNG_LIBS@
+JPWL_j2k_to_image_LDADD = $(top_builddir)/libopenjpeg/jpwl/libopenjpeg_JPWL.la  @LCMS1_LIBS@ @LCMS2_LIBS@ @TIFF_LIBS@ @PNG_LIBS@ -lm
 JPWL_j2k_to_image_SOURCES = \
 ../common/color.c \
 ../common/opj_getopt.c \
@@ -96,7 +96,7 @@ JPWL_image_to_j2k_CPPFLAGS = \
 @TIFF_CFLAGS@ \
 @PNG_CFLAGS@
 JPWL_image_to_j2k_CFLAGS =
-JPWL_image_to_j2k_LDADD = $(top_builddir)/libopenjpeg/jpwl/libopenjpeg_JPWL.la @TIFF_LIBS@ @PNG_LIBS@
+JPWL_image_to_j2k_LDADD = $(top_builddir)/libopenjpeg/jpwl/libopenjpeg_JPWL.la @TIFF_LIBS@ @PNG_LIBS@ -lm
 
 JPWL_image_to_j2k_SOURCES = \
 ../common/opj_getopt.c \
